[2023-03-21 12:30:16,996] {taskinstance.py:877} INFO - Dependencies all met for <TaskInstance: First_dag_v3.third_task 2023-03-21T12:30:13.321292+00:00 [queued]>
[2023-03-21 12:30:17,017] {taskinstance.py:877} INFO - Dependencies all met for <TaskInstance: First_dag_v3.third_task 2023-03-21T12:30:13.321292+00:00 [queued]>
[2023-03-21 12:30:17,019] {taskinstance.py:1068} INFO - 
--------------------------------------------------------------------------------
[2023-03-21 12:30:17,023] {taskinstance.py:1069} INFO - Starting attempt 1 of 6
[2023-03-21 12:30:17,025] {taskinstance.py:1070} INFO - 
--------------------------------------------------------------------------------
[2023-03-21 12:30:17,084] {taskinstance.py:1089} INFO - Executing <Task(BashOperator): third_task> on 2023-03-21T12:30:13.321292+00:00
[2023-03-21 12:30:17,112] {standard_task_runner.py:52} INFO - Started process 68 to run task
[2023-03-21 12:30:17,141] {standard_task_runner.py:76} INFO - Running: ['airflow', 'tasks', 'run', 'First_dag_v3', 'third_task', '2023-03-21T12:30:13.321292+00:00', '--job-id', '8', '--pool', 'default_pool', '--raw', '--subdir', 'DAGS_FOLDER/first_dag.py', '--cfg-path', '/tmp/tmpdarf_ovu', '--error-file', '/tmp/tmpagozvrm0']
[2023-03-21 12:30:17,147] {standard_task_runner.py:77} INFO - Job 8: Subtask third_task
[2023-03-21 12:30:17,223] {logging_mixin.py:104} INFO - Running <TaskInstance: First_dag_v3.third_task 2023-03-21T12:30:13.321292+00:00 [running]> on host 8c52af15d172
[2023-03-21 12:30:17,310] {taskinstance.py:1283} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=Mario
AIRFLOW_CTX_DAG_ID=First_dag_v3
AIRFLOW_CTX_TASK_ID=third_task
AIRFLOW_CTX_EXECUTION_DATE=2023-03-21T12:30:13.321292+00:00
AIRFLOW_CTX_DAG_RUN_ID=manual__2023-03-21T12:30:13.321292+00:00
[2023-03-21 12:30:17,314] {bash.py:135} INFO - Tmp dir root location: 
 /tmp
[2023-03-21 12:30:17,320] {bash.py:158} INFO - Running command: echo Im the third task and ill execute with the task2
[2023-03-21 12:30:17,368] {bash.py:169} INFO - Output:
[2023-03-21 12:30:17,403] {bash.py:173} INFO - Im the third task and ill execute with the task2
[2023-03-21 12:30:17,412] {bash.py:177} INFO - Command exited with return code 0
[2023-03-21 12:30:17,547] {taskinstance.py:1192} INFO - Marking task as SUCCESS. dag_id=First_dag_v3, task_id=third_task, execution_date=20230321T123013, start_date=20230321T123016, end_date=20230321T123017
[2023-03-21 12:30:17,721] {taskinstance.py:1246} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2023-03-21 12:30:17,832] {local_task_job.py:146} INFO - Task exited with return code 0
